tests/test_scenarios.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+import logging
+from pathlib import Path
+
+import pytest
+import requests
+from conftest import TEST_PATH
+from pyrdfj2 import J2RDFSyntaxBuilder
+from pyrdfstore import RDFStore, create_rdf_store
+from rdflib import Graph
+from util4tests import run_single_test
+
+from travharv import TravHarv
+from travharv.store import RDFStoreAccess
+
+log = logging.getLogger(__name__)
+
+QUERY_BUILDER: J2RDFSyntaxBuilder = J2RDFSyntaxBuilder(
+    templates_folder=str(Path(__file__).parent / "scenarios" / "templates")
+)
+
+BASE = "http://localhost:8080/"
+OUTPUTS = TEST_PATH / "output"
+INPUT = TEST_PATH / "input"
+CONFIGS = TEST_PATH / "travharv_config"
```

Comment:
``` python
CONFIGS: Path = TEST_Path / "travharv_config" 
```
add type-hints in python to all vars
good reminder to self + vscode will do a better job in code-completion-assistance